### PR TITLE
Toast for capture / upgrade outposts

### PR DESCRIPTION
**Pull request by grilledham**
Tested by myself in multiplayer

### CODE REVIEW

File: map_gen/presets/crash_site/outpost_builder.lua [hunk 1]
```diff
@@ -1521,7 +1535,7 @@ function Public.do_random_loot(entity, weights, loot)
     end
 
     entity.operable = false
-    entity.destructible = false
+    --entity.destructible = false
```
Comment (linaori) [inline map_gen/presets/crash_site/outpost_builder.lua:1538]: on purpose?

**Review by linaori — APPROVED** [review]
Looks good!

File: map_gen/presets/crash_site/outpost_builder.lua (hunk 1 above)
Comment (grilledham) [reply map_gen/presets/crash_site/outpost_builder.lua:1538]: yeah, really I should make a config file like diggy has, but that's a problem for future me

**Comment by grilledham** [comment]
I'm going to merge this so I can test plague's script to update the scenario on the server

**Comment by mheguy** [comment]
Failing travis' tests is an actual fail now. :p

**Pull request merged**